main.go

```diff
@@ -57,7 +58,8 @@ func process(fileName, app string, overwrite, defaultSelect, skipGenerated bool)
 		return err
 	}
 	if skipGenerated && ast.IsGenerated(file) {
-		return errors.New("skipping generated file")
+		log.Printf("skipping generated file: %s\r\n", fileName)
```

Comment:
maybe without \r please. do you need it for something specific? \n should do most cases, right?
```suggestion
		log.Printf("skipping generated file: %s\n", fileName)
```